src/ch.usi.inf.nodeprof/src/ch/usi/inf/nodeprof/ProfiledTagEnum.java

```diff
@@ -26,8 +26,8 @@
 public enum ProfiledTagEnum {
     UNARY(JSTags.UnaryOperationTag.class, -1), // have multiple case
     BINARY(JSTags.BinaryOperationTag.class, 2),
-    CF_COND(JSTags.ControlFlowBranchTag.class, 0), // to be checked
-    CF_BRANCH(JSTags.ControlFlowBlockTag.class, -1), // to be checked
+    CF_BRANCH(JSTags.ControlFlowBranchTag.class, -1), // to be checked
+    CF_BLOCK(JSTags.ControlFlowBlockTag.class, -1), // to be checked
```

Comment:
@Haiyang-Sun to issue the callback at the right time, the `-1` should be `0` here. I.e. we want to ignore the inputs to the block and issue the callback `onEnter` right away. (We don't have a use for the inputs, which I think are the statements, anyway.